request-manager/status/manager.go

```diff
@@ -89,9 +115,9 @@ func (m *manager) Running(f Filter) (proto.RunningStatus, error) {
 		seen[r.RequestId] = true
 	}
 
-	q := "SELECT request_id, type, state, user, created_at, started_at, finished_at, total_jobs, finished_jobs" +
+	q = "SELECT request_id, type, state, user, created_at, started_at, finished_at, total_jobs, finished_jobs" +
 		" FROM requests WHERE request_id IN (" + db.IN(ids) + ")"
-	rows, err := conn.QueryContext(ctx, q)
+	rows, err = conn.QueryContext(ctx, q)
```

Comment:
I'm not sure if/how this will work with the earlier `defer rows.Close()`.  I.e.:
```go
rows := .. // first val
defer rows.Close()
rows = ... // new val
defer rows.Close()
```
Not sure how Go handles that? Probably best to just avoid it by using `rows2` for the second call.